<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpgradeDialog</class>
 <widget class="QDialog" name="UpgradeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Upgrade...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxVersion">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="2">
       <widget class="QPushButton" name="m_btnRevert">
        <property name="text">
         <string>Revert</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="m_btnCheckVersion">
        <property name="text">
         <string>Check Version</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="m_btnUpgrade">
        <property name="text">
         <string>Upgrade</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="m_btnClose">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>51</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" rowspan="5">
       <widget class="QTableWidget" name="m_tabwdtVersionDetail">
        <property name="columnCount">
         <number>3</number>
        </property>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="m_lblVersionHint">
     <property name="text">
      <string>Your version is the latest.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
